src/web/app/components/teammates-common/notification-style-class.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { NotificationStyle } from '../../../types/api-output';
+import { notificationStyleConfigMap } from '../../../types/notification-style-config';
+
+/**
+ * Pipe to handle the transformation of an NotificationStyle to a css class.
+ */
+@Pipe({
+  name: 'notificationStyleClass',
+})
+export class NotificationStyleClassPipe implements PipeTransform {
+
+  transform(style: NotificationStyle): string {
+    return `bg-${style.toLowerCase()} ${notificationStyleConfigMap[style].font}`;
```

Comment:
Would it be possible to determine the colour of the font here instead of using an exported constant `notificationStyleConfigMap` in a new file? As currently the exported constant is only used in the two pipe files for two properties (description and font) without much reuse, so I am thinking if we could improve on this?

Reply:
I get what you mean. However, it is still shared by both files so it is necessary to be placed at somewhere else.

Of course, it can be placed within one of the pipes and the other one import it, but I think that is even more messy.